app/frontend/src/application.js

```diff
@@ -43,6 +44,7 @@ application.register('form', FormController);
 application.register('location-finder', LocationFinderController);
 application.register('manage-qualifications', ManageQualificationsController);
 application.register('map', MapController);
+application.register('sidebar', MapSidebarController);
```

Comment:
shouldn't this be
`application.register('map-sidebar', MapSidebarController);`
to be consistent with others?